### PR TITLE
Navigation.hid page

### PR DESCRIPTION
**Pull request by SnorlaxYum**
Added a page about this navigator property. And this page is needed for https://github.com/mdn/content/pull/4234 .
Compatibility PR: https://github.com/mdn/browser-compat-data/pull/10012
Still, the same problem as the parent doc: it seems the backend didn't recognize the correct specification url, it should be: https://wicg.github.io/webhid/ .

### CODE REVIEW

File: files/en-us/web/api/navigator/index.html [hunk 1]
```diff
@@ -35,6 +35,10 @@ <h3 id="Standard_properties">Standard properties</h3>
  <dd>Reports the value of the user's do-not-track preference. When this value is "yes", your web site or application should not track the user.</dd>
  <dt>{{domxref("Navigator.geolocation")}} {{readonlyInline}}</dt>
  <dd>Returns a {{domxref("Geolocation")}} object allowing accessing the location of the device.</dd>
+ <dt>{{domxref("Navigator.hid")}} {{readonlyInline}}</dt>
+ <dd>Returns an {{domxref("HID")}} object providing methods
+  for connecting to HID devices, listing attached HID devices and event
```
Comment (jpmedley) [inline files/en-us/web/api/navigator/index.html:40]:
```suggestion
  for connecting to HID devices, listing attached HID devices, and event
```

File: files/en-us/web/api/navigator/hid/index.html [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+- WebHID
+- WebHID API
+- Property
+- Reference
+---
+<div>{{APIRef("WebHID API")}}{{SeeCompatTable}}</div>
+
+<p><span class="seoSummary">The <code><strong>Navigator.hid</strong></code>
+    read-only property returns an {{domxref("HID")}} object providing methods
+    for connecting to HID devices, listing attached HID devices and event
```
Comment (jpmedley) [inline files/en-us/web/api/navigator/hid/index.html:16]:
```suggestion
    for connecting to HID devices, listing attached HID devices, and event
```

**Review by jpmedley — CHANGES_REQUESTED** [review]

**Review by jpmedley — APPROVED** [review]
Thank you for your help. If you're interested in doing more. I have some resources that can help.

**Comment by SnorlaxYum** [comment]
> Thank you for your help. If you're interested in doing more. I have some resources that can help.

@jpmedley ty. will certainly need ur help in my future PRs where we'll certainly meet again ;)

**Pull request merged**